src/containers/documentation/ApiPage.tsx

```
@@ -21,47 +21,56 @@ const DeactivationMessage = ({ api }: { api: IApiDescription }) => {
+    return null;
+  }
+
   const content = isDeactivated
-    ? api.deactivationInfo!.deactivationContent
-    : api.deactivationInfo!.deprecationContent;
+    ? api.deactivationInfo.deactivationContent
+    : api.deactivationInfo.deprecationContent;
   return (
     <div className={classNames('usa-alert', 'usa-alert-info', 'va-api-deprecation-alert')}>
       <div className={classNames('usa-alert-body')}>{content({})}</div>
     </div>
   );
 };
 
-export default class ApiPage extends React.Component<RouteComponentProps<IApiNameParam>> {
-  public render() {
-    const { params } = this.props.match;
-    const api = this.getApi();
-    if (api === null) {
-      return <ExplorePage />;
-    }
-
-    const category = lookupApiCategory(params.apiCategoryKey)!;
-    return (
-      <Flag name={`enabled.${api.urlFragment}`} fallbackComponent={ExplorePage}>
-        <div role="region" aria-labelledby={PAGE_HEADER_ID}>
-          <PageHeader halo={category.name} header={api.name} />
-          <DeactivationMessage api={api} />
-          {!isApiDeactivated(api) && (
-            <ApiDocumentation
-              apiDefinition={api}
-              categoryKey={params.apiCategoryKey}
-              location={this.props.location}
-            />
-          )}
-        </div>
-      </Flag>
-    );
+const getApi = (apiName?: string): IApiDescription | null => {
+  if (!apiName) {
+    return null;
   }
 
-  private getApi(): IApiDescription | null {
-    if (!this.props.match.params.apiName) {
-      return null;
-    }
+  return lookupApiByFragment(apiName);
+};
+
+const ApiPage = (props: RouteComponentProps<IApiNameParam>): JSX.Element => {
+  const { params } = props.match;
 
-    return lookupApiByFragment(this.props.match.params.apiName);
+  const api = getApi(params.apiName);
+  if (api === null) {
+    return <ExplorePage />;
   }
-}
+
+  const category = lookupApiCategory(params.apiCategoryKey);
+  if (category === null) {
+    return <ExplorePage />;
+  }
```

Added this for the same issue as above with non-null assertions. If category would be null, I don't know if we should return the explore page, if we should throw an error, or if we should set a default value and return the regular page.